Instagram/Views/CommentCell.h

```diff
@@ -0,0 +1,23 @@
+//
+//  CommentCell.h
+//  Instagram
+//
+//  Created by Megan Miller on 6/28/22.
+//
+
+#import <UIKit/UIKit.h>
+#import <Parse/Parse.h>
+#import "Comment.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface CommentCell : UITableViewCell
+
+@property (weak, nonatomic) IBOutlet UILabel *commentLabel;
+@property (nonatomic, strong) Comment *comment;
```

Comment:
This breaks MVC by connecting your view directly to the model. That's not necessarily a terrible thing to do, but it does make your view less reusable. Ideally you handle any data manipulation that has to happen outside of the view, and then you can simply set `commentLabel.text`.

Also consider that by exposing `commentLabel`, you open up the ability for anyone to change the settings of your label (i.e. the font, color, etc.) If you want to restrict these things, you can make the `commentLabel` private and expose only a `commentText` field. Part of programming is deciding how open or closed you want your APIs to be.